forage-android/src/main/java/com/joinforage/forage/android/collect/BTPinCollector.kt

```diff
@@ -22,10 +23,19 @@ import java.util.*
 
 internal data class ProxyRequestObject(val pin: TextElement, val card_number_token: String)
 
+internal data class POSProxyRequestObject(
+    val pin: TextElement,
+    val card_number_token: String,
+    val amount: String,
+    val reason: String,
+    val pos_terminal: Map<String, Any>,
```

Comment:
can we make the type for `pos_terminal` stricter? This format is also used by the proxy `/balance` endpoint:

```
{
  pos_terminal: {
    provider_terminal_id: String
  }
}
```